packages/destination-actions/src/destinations/stackadapt-audiences/forwardProfile/functions.ts

```diff
@@ -1,8 +1,52 @@
 import { RequestClient } from '@segment/actions-core'
 import { Payload } from './generated-types'
 import { domain } from '..'
+import { createHash } from 'crypto'
+
+const EXTERNAL_PROVIDER = 'segmentio'
+
+const standardFields = new Set([
+  'email',
+  'firstName',
+  'lastName',
+  'phone',
+  'emailMarketingStatus',
+  'company',
+  'gender',
+  'city',
+  'state',
+  'country',
+  'timezone',
+  'postalCode',
+  'birthDay',
+  'birthMonth',
+  'address',
+  'previousId'
+])
+const audienceMapping = stringifyJsonWithEscapedQuotes([
```

Comment:
Will we always only have those 2 attributes for audience or we could have more? These 2 should be sufficient for this iteration of our cdp project but just curious if we could have more

Reply:
I don't think there are more fields, but what additional fields do you think we might be looking for?

Reply:
like audience conditions for example: age from 12 to 24, or employees in certain company

Reply:
I can ask Joe but I'm doubtful that information is forwarded to destinations.